### DISCUSSION TITLE
Secondary button color

### DISCUSSION THREAD

**Discussion by frankfzhao** [twilio-labs/paste#1460] [Q&A] [answered]
I have noticed that there are some recent updates to the secondary button color. It changes from blue to gray border and text. Could someone clarify the reason making those changes? And I have seen in the Paste website there are some secondary button still using blue color, is that intentional? Thanks

**Comment by frankfzhao** [comment]
![image](https://user-images.githubusercontent.com/82524550/118564839-47728c00-b72e-11eb-9107-992bd1452110.png)
This is an example page in the figma component library

**Comment by frankfzhao** [comment]
![image](https://user-images.githubusercontent.com/82524550/118564873-62450080-b72e-11eb-9346-011a017035c1.png)
And this is a screenshot from Paste website

**Comment by serifluous** [comment, marked as answer]
Hey @frankfzhao , thanks for the question!

@katieporter sent an email out about this on May 5. Here's the snippet from the email:

>...we’re updating the look of secondary buttons! Secondary buttons will now have black text and a gray border. This will help give more visual prominence to primary buttons and reduce the blue overload that exists in some Twilio UIs today. Big thanks to the DSYS committee for all of the feedback as we worked on this change! This update will roll out in Figma this week and in the coded Button component within the next few weeks.

For more background context, we've received feedback from product designers over the past year that our secondary buttons are a bit too loud. The strong blue can make it harder for other components to stand out, especially components like Anchors when they might actually be a more important call-to-action than the secondary button.

In Figma, the secondary button currently has the "(dev)" label, which means we're still in the process of updating on the code side, so the update will come through the Paste site and the products soon.

Let us know if you have any concerns!
